cleaners/internet_explorer.xml

```diff
@@ -12,34 +12,43 @@
 
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.
 
     You should have received a copy of the GNU General Public License
-    along with this program. If not, see <http://www.gnu.org/licenses/>.
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
     @app Microsoft Internet Explorer
     @url https://www.microsoft.com/en-us/download/internet-explorer.aspx
     @os Windows
     @cleanerversion v2.0.0
-    @cleanerdate 2019-03-13
+    @cleanerdate 2019-03-17
     @cleanerby ??? & https://github.com/Tobias-B-Besemer
     @tested ok v???, Windows 7
-    @testeddate 2019-03-10
+    @testeddate 2019-03-10 (... with an old version of the cleaner!)
     @testedby https://github.com/Tobias-B-Besemer
     @note 
 
 -->
 <cleaner id="internet_explorer" os="windows">
   <label>Internet Explorer</label>
   <description>Web browser</description>
+  <var name="IELocalAppData">
+    <!-- Windows 7 Support (and maybe other Windows versions): -->
+    <value search="glob">%LocalAppData%\Microsoft\Windows</value>
+    <!-- Used e.g. in Windows XP (and maybe before on NT) with e.g. Internet Explorer 8 - English-only-Support: -->
+    <value search="glob">%UserProfile%\Local Settings</value>
+    <!-- And now some none-English-Support... -->
+    <!-- Used e.g. in Windows XP (and maybe before on NT) with e.g. Internet Explorer 8 - Polish-Support: -->
+    <value search="glob">%UserProfile%\Ustawienia lokalne</value>
+  </var>
   <option id="cookies">
     <label>Cookies</label>
     <description>Delete cookies, which contain information such as web site preferences, authentication, and tracking identification</description>
-    <action command="delete" search="walk.files" path="%USERPROFILE%\Cookies\"/>
-    <action command="delete" search="walk.files" path="%APPDATA%\Microsoft\Windows\Cookies\"/>
+    <action command="delete" search="walk.files" path="%UserProfile%\Cookies\"/>
+    <action command="delete" search="walk.files" path="%AppData%\Microsoft\Windows\Cookies\"/>
     <action command="delete" search="walk.files" path="%LocalAppData%\Microsoft\Internet Explorer\DOMStore\"/>
-    <action command="delete" search="walk.files" path="%LocalAppData%\Microsoft\Windows\INetCookies\"/>
+    <action command="delete" search="walk.files" path="$IELocalAppData\INetCookies\"/>
```

Comment:
Line 51 should be `$$IELocalAppData$$` instead of `$IeLocalAppData`?

Reply:
Yes, sorry! I fix that!